### PR TITLE
#452 add package metadata

### LINKED ISSUE #452: [FEATURE REQUEST]: Improve nuget package information
### Is your feature request related to a problem? Please describe.

I want to have a rich  as possible developer experience which means providing additional meta data as part of nuget package

### Describe the solution you'd like

The licence expression and repo url should be set

### Describe alternatives you've considered

Accept info is missing

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by thompson-tomo**
Closes #452

### CODE REVIEW

File: csharp/ParquetSharp.csproj [hunk 1]
```diff
@@ -19,16 +19,13 @@
     <Description>ParquetSharp is a .NET library for reading and writing Parquet files. It's implemented in C# as a PInvoke wrapper around apache-parquet-cpp to provide high performance and compatibility.</Description>
     <Copyright>Copyright G-Research 2021</Copyright>
     <PackageProjectUrl>https://github.com/G-Research/ParquetSharp</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/G-Research/ParquetSharp.git</RepositoryUrl>
     <PackageTags>apache parquet gresearch g-research .net c#</PackageTags>
-    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+    <PackageLicenseExpression>apache-2.0</PackageLicenseExpression>
```
Comment (adamreeve) [inline csharp/ParquetSharp.csproj:24]: This is case sensitive, it should be:
```suggestion
    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

File: csharp/ParquetSharp.csproj [hunk 2]
```diff
@@ -19,16 +19,13 @@
...
     <PackageOutputPath>..\nuget</PackageOutputPath>
   </PropertyGroup>
 
   <ItemGroup Label="Additional nuget files">
-    <None Include="..\LICENSE.txt">
```
Comment (adamreeve) [inline csharp/ParquetSharp.csproj:28]: I think this should be left so we continue to distribute a copy of the license in the NuGet package.

Comment (thompson-tomo) [reply csharp/ParquetSharp.csproj:28]: added back in

File: csharp/ParquetSharp.csproj (hunk 1 above)
Comment (adamreeve) [reply csharp/ParquetSharp.csproj:24]: This got reverted back to apache-2.0 somehow so the build is failing again

**Review by adamreeve — APPROVED** [review]
:+1:

**Pull request merged**